libfreerdp/core/transport.c

```diff
@@ -1002,8 +1150,9 @@ int transport_check_fds(rdpTransport* transport)
 	int status;
 	int recv_status;
 	wStream* received;
-	UINT64 now = GetTickCount64();
-	UINT64 dueDate = 0;
+//	UINT64 now = GetTickCount64();
```

Comment:
you just got rid of the timeout for network loss...

Reply:
i am a bit confused by this logic, can you please explain it in more details, as i understand it is a single threaded code so it is useless to use event here, am i wrong ?
second, about this timer, what does it do ?, as you can see, i have changed logic a bit, so we are not trying to read  whole pdu, instead we filling pdu buffer until at list one complete pdu in and after this calling data handler for one complete pdu, does timer really necessary with this design ?

Reply:
Implement a network timeout, as in if you do not receive anything within a defined amount of time assume the network is dead